README.md

```diff
@@ -91,14 +91,22 @@ need funded accounts for interacting with EVM._
 
 **With Docker**
 
-Run the following commands:
+Set the following ENV variables 
 
 ```bash
-cd dev
+FLOW_NETWORK_ID=[flow-testnet | flow-mainnet]
+ACCESS_NODE_GRPC_HOST=
+INIT_CADENCE_HEIGHT=211176670 
```

Comment:
This is correct for `testnet`, but I think it's a different value for `mainnet`. Maybe we should point this out too.

Reply:
Yeah, I maybe should have put the PR as draft :)

Reply:
No worries :sweat_smile: